### ISSUE TITLE
Black screen when running 2D platformer demo in linux

### ISSUE THREAD

**Issue by ombre5733** [AtomicGameEngine/AtomicGameEngine#1465]
I have built the AGE from source on an Ubuntu 16.10 system. Running the 2D platformer demo results in an empty (black) window. Since there is a sound when jumping, I guess that something happens with the display output.

**Comment by JimMarlowe** [comment]
In the AtomicEditor main menu, got to Tools->Logs->Player Log, then open AtomicPlayer.log. Are there any errors listed in the logfile?

**Comment by ombre5733** [comment]
There are no errors in AtomicPlayer.log. If needed, I can upload the whole log. 

The CEF.log in the same directory shows these errors:

```
[0307/224712:ERROR:browser_main_loop.cc(231)] Running without the SUID sandbox! See https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the sandbox on.
[0307/224712:ERROR:socket_posix.cc(124)] bind() returned an error, errno=98: Die Adresse wird bereits verwendet
[0307/224712:ERROR:devtools_http_handler.cc(228)] Cannot start http server for devtools. Stop devtools.
```

But I guess they are not related to the black screen.

**Comment by JimMarlowe** [comment]
Those CEF errors will not impact the display. CEF does run without the SUID set, and the physics 2d platformer is not using the the webview. The bind error is from running a CEF instance in the editor, and starting up a 2nd instance in the player, and wants to use the same port address.
There have been problems in the past with incompatible ALSA drivers causing the SDL initialization to fail, the way you can test this, is by setting "export SDL_AUDIODRIVER=auto", keeping in mind that the player may lose sound, but gain visuals.

**Comment by ombre5733** [comment]
Using "export SDL_AUDIODRIVER=auto" does not solve the black screen but I also lose the sound.

**Comment by JimMarlowe** [comment]
When I play this example, it starts out with a window that has you select the time of day, are you seeing that?

**Comment by ombre5733** [comment]
Yes it does so. I can then select to play daytime or nighttime. After the selection, the screen stays black. However, I can move around and hear sounds from jumping or collecting coins. By now, I wander if the camera is set up correctly.

**Comment by ombre5733** [comment]
The platformer works when I use "export LC_NUMERIC=C" before starting the engine (my default locale is "de_AT.UTF-8"). I only found this solution because it was mentioned in #710.

**Comment by JoshEngebretson** [comment]
Closed via #1466

**Comment by ombre5733** [comment]
Actually, #1466 does not fix this issue. The problem is that XML loading has the same problem. See #1469 for a follow-up issue.

**Comment by ombre5733** [comment]
Close via #1479.
